packages/web/pages/index.tsx

```diff
@@ -34,7 +108,24 @@ const Home: NextPage = observer(function () {
             />
           </g>
         </svg>
-        <TradeClipboard className="absolute left-[min(calc(920*(100vh/1080)),calc((100vw_-_206px)*0.8_-_520px))]" />
+      </div>
+      <div className="absolute w-full h-full flex items-center overflow-x-hidden overflow-y-auto">
+        <TradeClipboard
+          ref={clipboardRef}
+          containerClassName="w-full max-w-[32.5rem]"
+          containerStyle={{
+            ["--clipboardMinLeft" as any]: `calc(${clipboardMinLeftPx} * (100vh / 1080))`,
```

Comment:
Is tailwind not able to achieve this justification?

Reply:
Let's leave this problem for latter discussions.